metadata-integration/java/datahub-client/src/main/java/datahub/client/kafka/KafkaEmitter.java

```diff
@@ -37,7 +37,7 @@ public class KafkaEmitter implements Emitter {
   private static final int ADMIN_CLIENT_TIMEOUT_MS = 5000;
 
   /**
-   * The default constructor, prefer using the `create` factory method.
```

Comment:
Confirm if this is necessary change...

Reply:
Yes. As there is no create method in kafka emitter